debian/changelog

```diff
@@ -1,6 +1,5 @@
-nicotine (1.4.2-1) eoan; urgency=medium
+nicotine (1.4.3) eoan; urgency=medium
```

Comment:
Please don't modify the previous package changelog entry. Please add a new one via `dch -i`. The package version should probably be `1.4.3-1` The trailing 1 is the version of the packaging itself. So if we modify the Debian metadata or scripts in some way without modifying the "upstream" source, then we usually bump the package version.

Reply:
I used `dch`, now there is the value UNRELEASED in the distribution field, which I don't think is intended. But I don't think eoan is the correct value for it either, right? I leave it up to the you as the maintainer to decide the value.

Reply:
It's ok, `eoan` or actually better `focal` is fine since the PPA's build recipe will automatically substitute it for all of the different releases (e.g. disco, eoan, focal, etc.).